requirements.txt

```diff
@@ -1,6 +1,6 @@
 dotscience==0.8.0
 jupyterlab==1.0.9
-jupyterlab-dotscience-backend==0.2.19
+jupyterlab-dotscience-backend==0.2.24
```

Comment:
Is this meant to be a different version to the one above?

Reply:
Fixed.